.github/workflows/build.yml

```
@@ -52,5 +52,8 @@ jobs:
             exit 1
           fi
 
+          git config user.name "gotifyci"
+          git config user.email "gotifyci@gotify.net"
```

```suggestion
          git config user.email "gotify@protonmail.com"
```
this mail is registered to @gotifyci